src/Task/Category/CreateUpdateEntityTask.php

```diff
@@ -83,68 +61,12 @@ public function __invoke(PipelinePayloadInterface $payload): PipelinePayloadInte
-
-                $this->assignParent($taxon, $taxons, $resource);
-
-                foreach ($this->syliusAkeneoLocaleCodeProvider->getUsedLocalesOnBothPlatforms() as $syliusLocale) {
-                    $akeneoLocale = $this->syliusAkeneoLocaleCodeProvider->getAkeneoLocale($syliusLocale);
-
-                    $label = \sprintf('[%s]', $resource['code']);
-
-                    if (array_key_exists($akeneoLocale, $resource['labels']) && null !== $resource['labels'][$akeneoLocale]) {
-                        $label = $resource['labels'][$akeneoLocale];
-                    }
-
-                    $taxonTranslation = $this->taxonTranslationRepository->findOneBy([
-                        'translatable' => $taxon,
-                        'locale' => $syliusLocale,
-                    ]);
-
-                    if (!$taxonTranslation instanceof TaxonTranslationInterface) {
-                        /** @var TaxonTranslationInterface $taxonTranslation */
-                        $taxonTranslation = $this->taxonTranslationFactory->createNew();
-                        $taxonTranslation->setLocale($syliusLocale);
-                        $taxonTranslation->setTranslatable($taxon);
-                        $this->entityManager->persist($taxonTranslation);
-
-                        $this->logger->notice('Created TaxonTranslation', [
-                            'taxon_id' => $taxon->getId() ?? 'unknown',
-                            'taxon_code' => $taxon->getCode(),
-                            'locale' => $syliusLocale,
-                            'akeneo_locale' => $akeneoLocale,
-                        ]);
-                    }
-
-                    $taxonTranslation->setName($label);
-                    $slug = Transliterator::transliterate(
-                        str_replace(
-                            '\'',
-                            '-',
-                            sprintf(
-                                '%s-%s',
-                                $resource['code'],
-                                $label,
-                            ),
-                        ),
-                    );
-                    $taxonTranslation->setSlug($slug);
-
-                    $this->logger->notice('Update TaxonTranslation', [
-                        'taxon_id' => $taxon->getId() ?? 'unknown',
-                        'taxon_code' => $taxon->getCode(),
-                        'locale' => $syliusLocale,
-                        'akeneo_locale' => $akeneoLocale,
-                        'name' => $label,
-                        'slug' => $slug,
-                    ]);
-
-                    $this->handleAttributes($taxon, $resource);
-                }
+                $this->processorChain->chain($taxon, $resource);
 
                 $this->dispatcher->dispatch(new AfterProcessingTaxonEvent($resource, $taxon));
 
                 $this->entityManager->flush();
+                $this->sortableManager->enableSortableEventListener();
```

Comment:
Can you move enable/disable of Doctrine event arroud le foreach ? To have only on call for all Categories.